README.md

```diff
@@ -0,0 +1,9 @@
+You need create a new view from database AdventureWorks
```

Comment:
If you need to include a new SQL element such as a table, view, or stored procedure, it's recommended to create a new folder called 'Scripts', 'DbElements', etc. Using the 'Readme' file for this purpose can expose your internal queries to users, which is not ideal.